modules/tinymce/src/core/test/ts/browser/EditorSizeTest.ts

```diff
@@ -0,0 +1,76 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Height } from '@ephox/sugar';
+import { TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.EditorSizeTest', () => {
+  context('Minimum Size Enforcement', () => {
+    context('Unitless wrapper', () => {
+      const hook = TinyHooks.bddSetup<Editor>({
+        selector: 'textarea',
+        height: 10,
+        min_height: 500,
+        base_url: '/project/tinymce/js/tinymce'
+      }, []);
+      it('TINY-11108: unitless', () => {
+        const editor = hook.editor();
+        assert.equal(Height.get(TinyDom.container(editor)), 500);
+      });
+    });
+
+    context('em Wrapper', () => {
+      const hook = TinyHooks.bddSetup<Editor>({
+        selector: 'textarea',
+        height: '10em',
+        min_height: 500,
+        base_url: '/project/tinymce/js/tinymce'
+      }, []);
+      it('TINY-11108: using em units', () => {
+        const editor = hook.editor();
+        assert.equal(Height.get(TinyDom.container(editor)), 500);
+      });
+    });
+
+    context('pt Wrapper', () => {
+      const hook = TinyHooks.bddSetup<Editor>({
+        selector: 'textarea',
+        height: '10pt',
+        min_height: 500,
+        base_url: '/project/tinymce/js/tinymce'
+      }, []);
+      it('TINY-11108: using pt units', () => {
+        const editor = hook.editor();
+        assert.equal(Height.get(TinyDom.container(editor)), 500);
```

Comment:
This doesn't look right it looks like the 10pt value is just ignored and it clamps it to the min_height of 500px also. I think you might need to setup a dom structure then render the editor on that structure if you want to make sure that the em calculation is correct for the target.

- [ ] Make sure the tests actually test what is supposed to test

Reply:
Well, that is what we want? We want to set the height to 10pt, it is less than 500, so it's set to 500. That is the entire use of the min-value.

Reply:
🤦‍♂️  Missed the `Minimum Size Enforcement` context. You are right.